Lib/test/test_largefile.py

```diff
@@ -200,6 +213,7 @@ def run(sock):
         self.thread.start()
         event.set()
 
+    @skip_no_disk_space(TESTFN, size * 3)
```

Comment:
Same question here :-)

Reply:
Would you mind to copy/paste the command here (and also replace `size * 3` with `size * 2.5`)?